src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -69446,24 +69501,32 @@ public void BinaryOperator_15_WithDisallowNull()
 struct S
 {
     public static S? operator+(S? a, [DisallowNull] S? b) => throw null!;
-    static void F(S? x, S? y)
+    static void F1(S? x, S? y)
     {
         if (x == null) throw null!;
         S? s;
         s = x + x;
         s = x + y; // 1
         s = y + x;
+        s = y + y; // no warning
```

Comment:
If I understood properly, the key here is that we have a side-effect from line above that produced warning number 1. Although I think it's good to test that scenario (since it was affected by change), I think the intent of the test was to check each binary operation in isolation. 

Feels like we should split the binary operations to independent methods to ensure isolation.
If the purpose is to validate the nullable state of a variable as a side-effect of a binary operation, then doing `y.Value.ToString()` seems more direct.